dataRetriever/resolvers/topicResolverSender/diffPeerListCreator.go

```diff
@@ -73,6 +73,10 @@ func (dplc *DiffPeerListCreator) IntraShardPeerList() []core.PeerID {
 	return dplc.messenger.ConnectedPeersOnTopic(dplc.intraShardTopic)
 }
 
+func (dplc *DiffPeerListCreator) FullHistoryList() []core.PeerID {
```

Comment:
please add comments to exported methods.

Reply:
Done